packages/node-cli/README.md

```diff
@@ -25,9 +25,7 @@ Each version of CLI for Node.js in NPM works optimally with a specific NetSuite
 
 | Version in NPM | Version in NetSuite |
 |:--------------:|:-------------------:|
-| 1.2.X | 2021.1 |
-| 1.1.X | 2020.2 |
-| 1.0.X | 2020.1 |
+| 1.3.0  |       | 2021.2
```

Comment:
I think 2021.2 should be within the pipes :)

Reply:
And it should be on the top.

Reply:
Yes, now I see. It appeared on the desired location, or this is what I thought. I will check it ASAP.

Reply:
I think it is fine now, at least I see correct details in my Preview.